### PR TITLE
Add first tui init block

### PR DESCRIPTION
**Pull request by VanGrx**
This change just creates a base for using tui library.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -1,3 +1,69 @@
...
+                                    },
+                                    _ => {},
+                                }
+                            }
+                        },
+                        _ => {},
+                    }
+                }
+                }
+    }
```
Comment (matheusfigueredo) [inline src/main.rs]: I think you just forgot to run the fmt here :)

**Review by matheusfigueredo — APPROVED** [review]
Nice!

I just left a small comment, but I don't think it matters (now) :)

**Review by matheusfigueredo — APPROVED** [review]
LGTM! 🚀

File: src/main.rs (hunk 1 above)
Comment (VanGrx) [reply src/main.rs]: ... and indeed I did!

**Pull request merged**